docs/source/release/index.rst

```diff
@@ -11,7 +11,7 @@ Release Notes
 
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
-
+* :feature:`2839`: Add `group_concat` operation for Clickhouse backend
```

Comment:
Can you please leave a blank line between the previous paragraph and the list. I don't think this will render correctly in the html.